File: perl/fizzbuzz.pl
use v5.10;

for my $i (1..100) {
    if ($i % 15 == 0) {
        say "FizzBuzz";
    } elsif ($i % 3 == 0) {
        say "Fizz";
    } elsif ($i % 5 == 0) {
        say "Buzz";
    } else {
        say $i;
    }
}

File: julia/fizzbuzz.jl
function fizzbuzz(n)
    if n % 15 == 0
        return "FizzBuzz"
    elseif n % 3 == 0
        return "Fizz"
    elseif n % 5 == 0
        return "Buzz"
    else
        return string(n)
    end
end

for i in 1:100
    println(fizzbuzz(i))
end
